tools/elastic/migration/Migrations/1.0.8/up/post/00-update-unpublished_content.json

```diff
@@ -0,0 +1,13 @@
+{
+  "action": "updateMapping",
+  "settings": {
+    "indexName": "unpublished_content"
+  },
+  "data": {
```

Comment:
I think ideally we would do a full reindex.  I may still choose to do this tomorrow morning.